schematic/store/synapse.py

```diff
@@ -615,7 +630,14 @@ def _get_file_entityIds(self, dataset_files: List,  only_new_files: bool = False
             
             # find new files (that are not in the current manifest) if any
             for file_id, file_name in dataset_files:
-                if not file_id in manifest["entityId"].values:
+                if "entityId" in manifest.columns:
```

Comment:
This logic here needs to be clean up. But i think it should be a separate PR to refactor this part